src/Workspaces/Core/Portable/Differencing/LongestCommonSubsequence.cs

```diff
@@ -190,6 +214,21 @@ internal void Initialize()
             private int Offset => _length / 2;
         }
 
+        protected static VStack CreateStack()
+            => new VStack(s_pool);
+
+        private static ObjectPool<VBuffer> CreatePool()
+            => new ObjectPool<VBuffer>(() => new VBuffer());
+
+        public static void ClearBufferPool()
+            => s_pool = CreatePool();
```

Comment:
so object pools already have a max number of eleemnts they hold onto.  given that, is tehre a lot of benefit to this step given that you now cap the largest buffers you're willing to pool?

Reply:
It might free a few MBs. Not sure if it's worth it.

Reply:
I'll remove this. We can add this if we see more memory problems in future.